portfolio/src/main/webapp/index.html

```diff
@@ -4,9 +4,10 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyC5cHwqHrGWuCjfMAIbonz6q8LjFOlp6Jg"></script>
```

Comment:
This is not a big problem. But if would like to publish your web site, you might need to take care of the security of your API key. Otherwise it might be abused by someone that saw the key in your source code. You can consider adding some API restrictions on the key to prevent abusing.

![image](https://user-images.githubusercontent.com/635513/87896347-8b4b2500-ca7a-11ea-872d-751e0a57aa89.png)

Reply:
Thanks for your reminder, I will add some restrictions to my API key.